Apsitvarkom.DataAccess/PollutedLocationDTOFileRepository.cs

```diff
@@ -46,7 +46,7 @@ public Task<IEnumerable<PollutedLocationDTO>> GetAllAsync()
     public async Task<IEnumerable<PollutedLocationDTO>> GetAllAsync(Location inRelationTo)
     {
         return from pollutedLocation in await GetAllAsync()
-               orderby inRelationTo.DistanceTo(_mapper.Map<Location>(pollutedLocation))
+               orderby inRelationTo.DistanceTo(_mapper.Map<PollutedLocation>(pollutedLocation))
```

Comment:
Variable name `pollutedLocation` is confusing here. Right now it seems like it's a `PollutedLocation` that gets mapped into the same `PollutedLocation`. It's really a DTO though, so maybe a better name would be `pollutedLocationDTO` or just `dto` etc.